charts/elasticsearch-efs/values-newdemo.yaml

```diff
@@ -0,0 +1,55 @@
+replicaCount: 1
+
+image:
+  repository: ""
+  pullPolicy: IfNotPresent
+  tag: "main-3"
+
+imagePullSecrets: []
+nameOverride: ""
+fullnameOverride: ""
+
+serviceAccount:
+  create: true
+  annotations: {}
+  name: ""
+
+podAnnotations: {}
+
+podSecurityContext: {}
+
+securityContext: {}
+
+service:
+  type: ClusterIP
+  port: 9200
+  port2: 9300
+
+ingress:
+  enabled: false
+  className: ""
+  annotations: {}
+  tls: []
+
+resources: 
+  limits:
+    memory: "12Gi"
+
+autoscaling:
+  enabled: false
+  minReplicas: 1
+  maxReplicas: 100
+  targetCPUUtilizationPercentage: 80
+
+nodeSelector: {}
+
+tolerations: []
+
+affinity: {}
+
+pvc:
+  esPvClaim:
+    storageClass: efs-es
```

Comment:
value here should be "elasticsearch-gp-elastic", instead of efs-es. Otherwise it will not match the storageclass setup